.github/configuration/dump-restore-test-configuration.yml

```diff
@@ -3,68 +3,110 @@ dump-restore-version: [[
     version: source.latest,
     dump-data-as: copy,
     dump-format: plain,
-    logical-database: null
+    logical-database: null,
+    type: full
   },
   {
     version: target.latest,
     dump-data-as: copy,
     dump-format: plain,
-    logical-database: null
+    logical-database: null,
+    type: full
+  }
+],
+[
+  {
+    version: source.latest,
+    dump-data-as: copy,
+    dump-format: plain,
+    logical-database: null,
+    type: schema_only
+  },
+  {
+    version: target.latest,
+    dump-data-as: copy,
+    dump-format: plain,
+    logical-database: null,
+    type: schema_only
```

Comment:
this is bit misleading, in case of schema only we are doing (schema only followed by data only), may be call it something else